src/vpnodes/basic/logic.ts

```diff
@@ -129,73 +129,3 @@ export class LogicScopeNode extends BlockNode {
     super.deserialize(sn);
   }
 }
-
-export class ConditionNode extends VPNode {
```

Comment:
As per usual: If something is unused and gets in the way, I prefer removing it over maintaining it. It's still there in the git history anyways.